pdf/delivery-slip.style-tab.tpl

```diff
@@ -121,16 +121,16 @@
 			border-top: 1px solid #000000;
 		}
 
-		.left {
-			text-align: left;
+		.text-start {
+			text-align: {$align_start};
 		}
 
-		.fright {
```

Comment:
I don't see any usage of this class in your PR neither in the project. Are you sure it is still necessary ? :thinking: